### PR TITLE
Add Calculator component

### PR DESCRIPTION
**Pull request by desorgui**
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

My component design :

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
 );
```
Comment (KDlamini) [inline src/index.js]: - Kindly refactor the ` index.js` to be the way that it was before. it must import the `App.js` component and render it to the ReactDOM.
- Only the `App.js` component should ever be rendered in the `index.js` file. All other components should be routed in the `App.js` main component.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi  @desorgui  🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating the Calculator component.
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Good PR title and dewscription
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
 - [ ] Please update your README and customize it to make it more relevant to the current project following the [microverse template](https://github.com/microverseinc/readme-template).

Since I cannot access your `App.js` file I'll leave the comments here.

- The `Calculator.js` component should be imported and rendered in `App.js`(main component).
- The `App.js` component should be class-based.
- All the boilerplate that got created by the CRA (texts, images, and styles) should be removed.
For example:
```
import React, { Component } from 'react';
import Calculator from ./components;

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by desorgui** [comment]
Thank you @KDlamini

**Review by descholar-ceo — APPROVED** [review]
Hi @desorgui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Addressed all requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**